doc/appendices/administration.rst

```diff
@@ -158,9 +158,10 @@ release. When Celeritas is declared stable, ``x.0.0`` is a major release,
 
 Major and minor releases (including 0.x.0 development releases) must have a
 milestone in the git issue tracker with a list of issues that can be assigned.
-Only major releases can deprecate or remove features and change
-:ref:`public-facing APIs <api>`. Both major and minor releases should include
-notable improvements to the code.
+Only major releases can remove features and change
+:ref:`public-facing APIs <api>`. Minor releases can
+:ref:`deprecate features <deprecations>`. Both major and minor releases should
+include notable improvements to the code.
```

Comment:
Above, we have that 0.x.0 is major and 0.x.z is patch (i.e., no minor releases while we are in development phase). This would seem to imply no deprecations while we are 0.x, which is at odds with the footnote in the deprecations section.

Reply:
I'd planned to try deprecating code in the development phase, but then decided it's too much to bother with... although I think I might follow the deprecation rules for public APIs in `accel`...

Reply:
I'm with you on that. Thanks for clarifying!